bpf/cpu/cpu.bpf.c

```diff
@@ -182,15 +192,23 @@ typedef struct {
   mapping_t mappings[MAX_MAPPINGS_PER_PROCESS];
 } process_info_t;
 
-// A row in the stack unwinding table for x86_64.
-typedef struct __attribute__((packed)) {
-  u64 pc;
-  u8 cfa_type;
-  u8 rbp_type;
-  s16 cfa_offset;
-  s16 rbp_offset;
-} stack_unwind_row_t;
-_Static_assert(sizeof(stack_unwind_row_t) == 14, "unwind row has the expected size");
+   // A row in the stack unwinding table for Arm64.
+  typedef struct __attribute__((packed)) {
+    u64 pc;
+    #if __TARGET_ARCH_arm64
+      s16 lr_offset;
+    #endif
+    u8 cfa_type;
+    u8 rbp_type;
+    s16 cfa_offset;
+    s16 rbp_offset;
+  } stack_unwind_row_t;
+  #if __TARGET_ARCH_arm64
+    _Static_assert(sizeof(stack_unwind_row_t) == 16, "unwind row has the expected size");
```

Comment:
nit: 
```suggestion
_Static_assert(sizeof(stack_unwind_row_t) == 16, "unwind row has the expected size");
```

`make bpf/fmt` should fix these!